js/_current-weather.js

```diff
@@ -127,22 +127,59 @@ async function displayCurrentWeatherData(data) {
   //     `
   // weatherDataElement.innerHTML = weatherInfo
   // Update City Header
+  console.log(data)
   const currentCity = document.getElementById('current-city-name');
   currentCity.innerHTML = data.name;
-
+  
   const currentTemp = document.getElementById('current-city-temp');
   currentTemp.innerHTML = `${convertKelvinToCelsius(data.main.temp)}°C`
 
+  const currentTempDetail = document.getElementById('current-weather-detailInfo-temperature-text');
+  currentTempDetail.innerHTML = `${convertKelvinToCelsius(data.main.temp)}°C`
+
+
     const dateTime = new Date(data.dt * 1000)
     const date = dateTime.toLocaleDateString('en-US', { month: 'short', day: 'numeric' })
     const dayName = dateTime.toLocaleDateString('en-US', { weekday: 'long' })
+    const weatherIcon = data.weather[0].icon;
+    const weatherDetail = data.weather[0].description;
+    
+    //rain calculation 
+    let rain;
```

Comment:
```suggestion
    let rain = '';
```
There is a best for initiation variables.
Even if it's not necessary to have a value at beginning, It'll be helpful and easier to read and understand source code.

Please take a look if you want.
https://www.w3schools.com/js/js_best_practices.asp